core/src/apps/ethereum/networks.py

```diff
@@ -23,6 +28,9 @@ def by_chain_id(chain_id: int) -> Optional["NetworkInfo"]:
 
 
 def by_slip44(slip44: int) -> Optional["NetworkInfo"]:
+    if slip44 == SLIP44_WANCHAIN:
+        # Coerce to Ethereum
+        slip44 == SLIP44_ETHEREUM
```

Comment:
just noting for myself that this typo (`==` instead of `=`) should cause *something* to fail. The question is what?

Reply:
Hmm, interesting ... will you fix master?

Reply:
it *would* fail if Ethereum was RSKIP60 compliant -- that's the only feature that is tested on the return value from this function.